SmartDeviceLinkTests/DevAPISpecs/SDLVideoStreamingRangeSpec.m

```diff
@@ -0,0 +1,221 @@
+                it(@"should be set to the minimum", ^{
+                    testRange.minimumDiagonal = -2.0;
+                    expect(testRange.minimumDiagonal).to(equal(defaultMinimumDiagonal));
+                });
+            });
+
+            context(@"above the minimum", ^{
+                it(@"should set the value", ^{
+                    testRange.minimumDiagonal = testMinimumDiagonal;
+                    expect(testRange.minimumDiagonal).to(equal(testMinimumDiagonal));
+                });
+            });
+        });
+    });
+
+    describe(@"checking if the resolution is in a given range", ^{
+        beforeEach(^{
+            testRange = [[SDLVideoStreamingRange alloc] initWithMinimumResolution:lowResolution maximumResolution:highResolution];
+        });
+
+        context(@"when there is no minimum or maximum resolution", ^{
+            beforeEach(^{
+                testRange = [[SDLVideoStreamingRange alloc] initWithMinimumResolution:nil maximumResolution:nil];
+            });
+
+            it(@"should return NO", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:2 height:2]]).to(beFalse());
+            });
+        });
+
+        context(@"when there is no maximum resolution", ^{
+            beforeEach(^{
+                testRange = [[SDLVideoStreamingRange alloc] initWithMinimumResolution:lowResolution maximumResolution:nil];
+            });
+
+            it(@"should return YES", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:2 height:2]]).to(beTrue());
+            });
+        });
+
+        context(@"when there is no minimum resolution", ^{
+            beforeEach(^{
+                testRange = [[SDLVideoStreamingRange alloc] initWithMinimumResolution:nil maximumResolution:highResolution];
+            });
+
+            it(@"should return YES", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:2 height:2]]).to(beTrue());
+            });
+        });
+
+        context(@"when the resolution is below the range", ^{
+            it(@"should return NO", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:0 height:0]]).to(beFalse());
+            });
+        });
+
+        context(@"when the resolution is above the range", ^{
+            it(@"should return NO", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:34463 height:34463]]).to(beFalse());
+            });
+        });
+
+        context(@"when the resolution is in the range", ^{
+            it(@"should return YES", ^{
+                expect([testRange isImageResolutionInRange:[[SDLImageResolution alloc] initWithWidth:2 height:2]]).to(beTrue());
+            });
+        });
+    });
+
+    describe(@"checking if the aspect ratio is in a given range", ^{
```

Comment:
Tests in this section are missing the expected result `to(beTrue())` or `to(beFalse())`

Reply:
Fixed